UI/Pages/Enquiry/Respond/Response.cshtml

```diff
@@ -2,204 +2,187 @@
-
-            @if (!string.IsNullOrWhiteSpace(Model.Data.EnquiryAdditionalInformation))
-            {
-                <div asp-validation-group-for="Data.AdditionalInformationText">
-                    <div class="govuk-form-group">
-                        <govuk-character-count asp-for="Data.AdditionalInformationText" threshold="@IntegerConstants.EnquiryQuestionsMaxCharacterThreshold" rows="12" max-length="@IntegerConstants.EnquiryQuestionsMaxCharacterSize">
-                            <govuk-character-count-label class="govuk-label--s" is-page-heading="true">
-                                Can you support those considerations?
-                            </govuk-character-count-label>
-                            <govuk-character-count-hint>
-                                Give as much detail as possible to increase your chance of matching the school’s tuition requirements
-                            </govuk-character-count-hint>
-                        </govuk-character-count>
-                    </div>
-                </div>
-            }
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h1 class="govuk-heading-m govuk-!-padding-top-0">
+          <span>Respond to this enquiry</span>
+        </h1>
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          <span>Key stage and subjects:</span>
+        </h2>
+
+        <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
+          @foreach (var item in Model.Data.EnquiryKeyStageSubjects)
+          {
+            <li>
+              @item
+            </li>
+          }
+        </ul>
+
+        <div asp-validation-group-for="Data.KeyStageAndSubjectsText">
+          <govuk-character-count asp-for="Data.KeyStageAndSubjectsText" threshold="@IntegerConstants.EnquiryQuestionsMaxCharacterThreshold" rows="12" max-length="@IntegerConstants.EnquiryQuestionsMaxCharacterSize">
+            <govuk-character-count-label class="govuk-label--s" is-page-heading="true">
+              Can you support those key stages and subjects?
+            </govuk-character-count-label>
+            <govuk-character-count-hint>
+              For example, you may be able to support all of these but not at the same time
+            </govuk-character-count-hint>
+          </govuk-character-count>
         </div>
-    </div>
-</div>
-    <input type="hidden" name="handler" value="Submit"/>
-    <input asp-for="Data.LocalAuthorityDistrict" type="hidden"/>
-    <input asp-for="Data.SupportReferenceNumber" type="hidden"/>
-    <input asp-for="Data.TuitionPartnerName" type="hidden"/>
-    @if (Model.Data.EnquiryKeyStageSubjects.Any())
-    {
-        @for (int i = 0; i < Model.Data.EnquiryKeyStageSubjects.Count(); i++)
+
+        <hr class="govuk-section-break govuk-section-break--visible govuk-!-padding-bottom-0 govuk-!-margin-top-0 govuk-!-margin-bottom-7">
+
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          Tuition type:
```

Comment:
It doesn't match the `Tuition of type:` we display for the enquirer view response page. It should be either `Tuition type:` or `Tuition of type:` for both pages to be consistent.

Reply:
Agreed - I have updated all that I could find to be "Type of tuition"